.changeset/hot-flowers-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+'legend-transactional': minor
```

Comment:
Is not a minor, is a patch.

Reply:
https://github.com/legendaryum-metaverse/legend-transactional/pull/149/commits/4daa97408d836948b23f5f600e509a346755340d

Reply:
Delete this file, **changeset** going to take the minor update